api_3/users_get_common.py

```diff
@@ -67,75 +42,33 @@ def get(self, request):
         if user_email is None:
             raise endpoints.UnauthorizedException("Authentication unsuccessful.")
 
-        # this checks the controlled-access google group
-        am_dbgap_authorized = is_dbgap_authorized(user_email)
-
-        if not am_dbgap_authorized:
+        das = DatasetAccessSupportFactory.from_webapp_django_settings()
+        authorized_datasets = das.get_datasets_for_era_login(user_email)
+        all_datasets = das.get_all_datasets_and_google_groups()
+        
+        authorized = False
+        allowed = False
+        for dataset in authorized_datasets:
+            if program in dataset:
+                authorized = True
```

Comment:
See the email I just sent - get_datasets and get_all_datasets return a specific kind of class called a DatasetGoogleGroupPair which has the ACL and the whitelist ID, but no program affiliation. You'll need to use the Django ORM for that.